src/binaryDecode.js

```diff
@@ -0,0 +1,5 @@
+// Prevent Closure from minifying the binaryDecode() function, or otherwise
+// Closure may analyze through the WASM_BINARY_DATA placeholder string into this
+// function, leading into incorrect results.
+/** @noinline */
+function binaryDecode(bin) { for(var i = 0, l = bin.length, o = new Uint8Array(l); i < l; ++i) o[i] = bin.charCodeAt(i) - 1; return o; }
```

Comment:
No need for this to be all on one line is there?

Seems a little unfortunate to create an entirely new file just to hold this tiny function.   Perhaps you can add it to `runtime_shared.js` instead?

Reply:
The file proxyClient.js does not include runtime_shared.js, so that would not quite work.

Expanded the function on multiple lines.